RealEstate/RealEstate/wwwroot/css/site.css

```diff
@@ -69,3 +69,17 @@ body {
   white-space: nowrap;
   line-height: 60px; /* Vertically center the text there */
 }
+
```

Comment:
// Change 3 - Table and Header Changes in Styles

Reply:
How does this align with accessibility?  Does the contrast meet the proper requirements?